content/corectl/_index.md

```diff
@@ -60,9 +60,17 @@ sudo mv ./corectl /usr/local/bin/corectl
 
 ### From source
 
-To install `corectl` from source, you need to have [Go](https://go.dev/learn/) installed.
+To install `corectl` from source, you need to have [Go](https://go.dev/learn/) and [golangci-lint](https://golangci-lint.run/) installed.
 
-```bash
+
+### Golangci-lint
+```sh
+# binary will be $(go env GOPATH)/bin/golangci-lint
+curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.61.0
```

Comment:
Do we really want to add a curl | sh in the docs here? Wouldn't it be sufficient to just mention that golangci-lint is required above, like we do for go?